### PR TITLE
exterminator: first pass at an exterminator CLI tool

### PR DESCRIPTION
**Pull request by nicks**
Hello @maiamcc,

Please review the following commits I made in branch nicks/ch5079/cli:

c26da4b7c797ff295f86d8c846520ddfcd675744 (2020-03-05 18:47:13 -0500)
exterminator: first pass at an exterminator CLI tool
This may eventually grow into a hosted service that we use to manage
exterminator duties, but for now you have to manage your own API keys.

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
Mostly did this in NodeJS because there are already pretty mature clients for both github and clubhouse in nodejs. Don't worry about the diff -- most of it is dependencies and argument parsing

File: index.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  console.error('Please set the CLUBHOUSE_API_TOKEN env variable')
+  process.exit(1)
+}
+
+const githubToken = process.env.GITHUB_API_TOKEN
+const octokitOptions = {}
+if (githubToken) {
+  octokitOptions.auth = githubToken
+} else {
+  console.warn('Warning: A GITHUB_API_TOKEN set in your environment will prevent you from getting rate-limited')
```
Comment (maiamcc) [inline index.js]: nit: direction of this warning is unclear. Maybe consider:

> Warning: no GITHUB_API_TOKEN found in your env. Consider setting one to keep you from getting rate-limited.

**Review by maiamcc — APPROVED** [review]
oooh neat!

File: index.js (hunk 1 above)
Comment (nicks) [reply index.js]: done!

**Pull request merged**